package.json

```diff
@@ -85,7 +88,8 @@
     "sass": "~1.32.13",
     "sass-loader": "^10.1.1",
     "sinon": "^14.0.0",
-    "typescript": "^3.9.10",
+    "ts-jest": "^27.0.4",
+    "typescript": "4.5.5",
```

Comment:
Updated the Typescript version to 4.5.5. I have been testing this with adding different dependencies one by one (so you will see some in here). Builds fine but Pinia is not working

![image](https://user-images.githubusercontent.com/116035339/235992132-2d295f29-3bf4-4c3c-a41a-0cc591a18983.png)

Reply:
Is this comment obsolete now?

Reply:
Yes. This is fixed